### ISSUE TITLE
Support for Python 2.5 has turned off

### ISSUE THREAD

**Issue by volym3ad** [twilio/stashboard#81]
[http://stashboard.appspot.com/](http://stashboard.appspot.com/)   
Please refer to https://goo.gl/aESk5L for more information

**Comment by codestro** [comment]
Yep, that's as of two days ago (June 20, 2017) that Google fully deprecated Python 2.5, which probably isn't a bad thing—can't hang on to 2011 forever! Your best bet will be to deploy it on Google Cloud in your own instance, but edit the app.yaml file to reference Python 2.7.13 instead, or any other Python 2.x version from 2.6.x forward. 

Judging by the last commit date on this project, Twilio no longer maintains stashboard, so there's no guarantee that the much newer Python 3.x will even be compatible with the app. https://wiki.python.org/moin/Python2orPython3 has more info on the specifics as for what's been deprecated since 2.x.

Best of luck!

**Comment by volym3ad** [comment]
Thanks.

**Comment by abitrolly** [comment]
Came to report the same error, but it looks like https://status.twilio.com/ still uses it.

**Comment by dnet5** [comment]
> Came to report the same error, but it looks like https://status.twilio.com/ still uses it.

Today Twilio's page is provided by [Statuspage](https://www.statuspage.io/) (from Atlassian).
![twilio](https://user-images.githubusercontent.com/3988029/50972109-de7b0980-14cc-11e9-88d0-ceec64301182.png)
